README.md

```diff
@@ -1,5 +1,8 @@
 # fuzzer-test-suite
 
+:zap: **NOTE: Fuzzer-test-suite has been replaced by [FuzzBench](https://github.com/google/fuzzbench). 
```

Comment:
Is it "replaced" completely?  Maybe we should be more specific.

e.g., For most use cases, fuzzer-test-suite is superseded by FuzzBench.  We recommend using FuzzBench for all future fuzzer benchmarking...

Reply:
Done!